python/ccxt/base/decimal_to_precision.py

```diff
@@ -54,9 +54,9 @@ def power_of_10(x):
     if precision < 0:
         to_nearest = power_of_10(precision)
         if rounding_mode == ROUND:
-            return str(to_nearest * decimal.Decimal(decimal_to_precision(dec / to_nearest, rounding_mode, 0, DECIMAL_PLACES, padding_mode)))
+            return str(to_nearest * decimal.Decimal(decimal_to_precision("{:f}".format(dec / to_nearest), rounding_mode, 0, DECIMAL_PLACES, padding_mode)))
```

Comment:
I meant instead of the call to `str`

Reply:
Can't remove it as we need to return string while to_nearest is decimal

Reply:
Use '{:f}'.format instead of `str` because str(0.000000000000002) will give scientific notation, same thing happens on very large numbers